src/hotspot/cpu/x86/x86_64.ad

```diff
@@ -7673,6 +7675,8 @@ instruct addI_rReg_mem(rRegI dst, memory src, rFlagsReg cr)
 %{
   match(Set dst (AddI dst (LoadI src)));
   effect(KILL cr);
+  flag(PD::Flag_sets_overflow_flag, PD::Flag_sets_sign_flag, PD::Flag_sets_zero_flag, PD::Flag_sets_carry_flag, PD::Flag_sets_parity_flag);
+
```

Comment:
Please remove these extra blank lines. On `addI_rReg_mem`, `addI_mem_rReg`, and `blsrL_rReg_rReg`.